generate.xml

```diff
@@ -235,7 +235,8 @@
       <build name="bitcoin" github="libbitcoin" repository="libbitcoin" branch="master" parallel="true" >
         <option value="${with_boost}" />
         <option value="${with_pkgconfigdir}" />
-        <!--<option value="--disable-silent-rules" />-->
+        <!-- Replace XX with two consecutive hyphens if not commented. Two consecutive hypens not allowed in comments -->
+        <!--<option value="XXdisable-silent-rules" />-->
```

Comment:
I'd prefer we just remove the offending commented out options. Otherwise this looks fine.

Reply:
Is this still active?

Reply:
Yes. So, should they be removed?

Reply:
Yes

Reply:
Status?